package.json

```diff
@@ -31,9 +31,6 @@
     "jit-grunt": "0.3.2",
     "time-grunt": "0.3.1"
   },
-  "peerDependencies": {
-    "grunt": "~0.4.0"
```

Comment:
Instead, upgrade to `>=0.4.0`. This will match most environments and not cause an error